packages/ses/scripts/hermes.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+OS="$(uname -s)"
+
+case "$OS" in
+    Linux*)
+        OS_DIR="linux64-bin"
+        ;;
+    Darwin*)
+        OS_DIR="osx-bin"
+        ;;
+    CYGWIN*|MINGW*|MSYS*)
+        OS_DIR="win64-bin"
+        ;;
+    *)
+        echo "Unsupported OS: $OS"
+        exit 1
+        ;;
+esac
+
+# Paths relative to 'packages/ses'
+HERMESC="../../node_modules/hermes-engine-cli/$OS_DIR/hermesc"
+HERMES="../../node_modules/hermes-engine-cli/$OS_DIR/hermes"
+
+echo "Catenating: dist/ses-hermes.cjs + test/hermes-smoke.js"
```

Comment:
```suggestion
echo "Concatenating: dist/ses-hermes.cjs + test/hermes-smoke.js"
```

I didn't find "catenating" in the dictionary, but [catenation](https://en.wikipedia.org/wiki/Catenation) is a chemistry thing I don't understand.

Reply:
thought i couldn't go wrong with verb form of [cat (Unix)](https://en.wikipedia.org/wiki/Cat_(Unix)), so it's clear it's unrelated to js `Array.prototype.concat()`

but you're right i made up a word